update(getting started): fixes of format, lg, and style issues (#33)
aiven/aiven-docs: docs/products/cassandra/get-started.md

---
title: Get started with Aiven for Apache Cassandra®
sidebar_labels: Get started
---

> Quick and easy way to start using Aiven for Apache Cassandra®: create a service either in the [Aiven Console](https://console.aiven.io/) or with the Aiven CLI.

Reviewer: That's not for us to decide :) Let's never judge complexity please.  (applies to all files of the PR)

```suggestion
Start using Aiven for Apache Cassandra® by creating a service either in the [Aiven Console](https://console.aiven.io/) or with the Aiven CLI.
```

## Create a service in the console

1.  Log in to the [Aiven Console](https://console.aiven.io/).

> 1.  Create an Aiven for Apache Cassandra® following the steps in
>     [Create a service](/docs/platform/howto/create_new_service).

Reviewer: Why not this directly, they know they have to follow instructions ⚡ 
```suggestion
1.  [Create an Aiven for Apache Cassandra®](/docs/platform/howto/create_new_service).
```

    Once the service is ready, its status changes to **Running**, which typically takes a
    couple of minutes, depending on your selected cloud provider and region.

## Create a service with the Aiven CLI

To create a service using the
[Aiven CLI](https://github.com/aiven/aiven-client), decide on the service plan, cloud
provider, and region for your new service and run the `avn service create` command.

```bash
avn service create demo-cassandra       \
   --service-type cassandra             \
   --cloud CLOUD_AND_REGION             \
   --plan PLAN                          \
   --project PROJECT_NAME
```

:::note
Preview the full list of default flags using the `avn service create -h`

> command. Additionally, there are some type-specific options, which you
> can see executing the `avn service types -v` command.

Reviewer: The grammar make it unclear what's `executing` things, is it `you` or the `options`, let's make it uninterpretable:

```suggestion
command. Additionally, list the type-specific options by running `avn service types -v`.
```
